### ISSUE TITLE
Tensorflow version of StarGANv2 three-four times slower than Pytorch version in training mode

### ISSUE THREAD

**Issue by baymak** [clovaai/stargan-v2-tensorflow#7]
This is not an issue but I am curious why there is such a big difference. Is it about implementation or difference btw Tensorflow vs Pytorch? thanks

**Comment by baymak** [comment]
I gets normal after a few hundred epochs
